content/en/serverless/step_functions/redrive.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Retry or redrive AWS Step Function executions
```

Comment:
The reason for the edit on this section is that `retry` for Step Functions is a complete separate concept from Redrive. We shouldn't mention it here since the term `retry` will confuse our customers.